### PR TITLE
Trigger digest loop on events

### PR DESCRIPTION
**Pull request by homerjam**

### CODE REVIEW

File: src/eval-service.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-'use strict';
```
Comment (homerjam) [inline src/eval-service.js:1]: Not used in ES6 AFAIK

File: src/eval-service.js [hunk 2]
```diff
@@ -9,11 +7,13 @@ class EvalService {
...
         };
 
-        if(options.attrs) {
+        if (options.attrs) {
             locals.attrs = options.attrs;
         }
 
-        scope.$eval(options.fn, locals);
+        scope.$apply(() => {
+            scope.$eval(options.fn, locals);
```
Comment (vernak2539) [inline src/eval-service.js:15]: Just wondering. What is the rationale behind this change?

**Review by vernak2539 — CHANGES_REQUESTED** [review]
Tests will need to be updated

File: src/eval-service.js (hunk 2 above)
Comment (homerjam) [reply src/eval-service.js:15]: The reason for this is to prevent the user having to do this themselves and to allow for the possibility of the following:

```html
<video ng-class="{fadeInNicely: vm.loaded}" on-loaded-metadata="vm.loaded = true" src="vid.mp4">

**Comment by homerjam** [comment]
Sorry did this in the browser, will try to update checks when I can

File: src/eval-service.js (hunk 2 above)
Comment (vernak2539) [reply src/eval-service.js:15]: Okay cool, that makes sense. Just need to make sure the old function way works, which it should, but just have to say it for the record!

**Comment by vernak2539** [comment]
@homerjam merged and published as v1.2.0. Thanks for the contribution!
